### ISSUE TITLE
About the performance of pretrained model

### ISSUE THREAD

**Issue by alan-ai-learner** [LynnHo/AttGAN-Tensorflow#48]
The pre-trained model you provided is not well-performing over the Celeb-A-HQ dataset. So I've got a question that for how many epochs you have trained the pre-trained model and on what data set. 
Another question is that my use case applies glasses to the face, so I need to know that if I trained a new model from scratch over the Celeb-A-HQ dataset it will help us to achieve my task.
can we train the model over a single attribute like eyeglasses or a smile?
Thanks in advance.

**Comment by LynnHo** [comment]
In the output folder, you can find `settings.yml`, there is the setting of the pre-trained model (200 epochs for CelebA-HQ dataset). Actually, this method does not perform well enough on eyeglasses. If you want a model that is better on a single and local attribute, I suggest SaGAN.

**Comment by alan-ai-learner** [comment]
Thanks for your reply.
As you were suggesting the SaGAN, I'll take a look at it.
By the way, my use case is I'm gonna pass a face image to the GAN model and it should return the same image with eyeglasses.
--------------------------------------
I'm going through all the issues of this repo ill found that for testing any image we need to pass the label attributes corresponding to the same image as you suggest that i looked into the STGAN repo for attribute classification. I was looking at it but haven't found a way to do it I think the code was hardcoded for a CelebA dataset. Is there any way to do attribute classification for a custom image?

**Comment by alan-ai-learner** [comment]
I looked at SaGAN(Slf Attention GAN) but not found a way to do my job, because the whole code written about training the dataset. But there is no mention for what purpose it actually used for.

**Comment by LynnHo** [comment]
I mean this paper, Zhang, Gang, et al. "Generative adversarial network with spatial attention for face attribute editing." Proceedings of the European conference on computer vision (ECCV). 2018.

Or you can refer to other solutions like https://github.com/genforce/interfacegan.

**Comment by alan-ai-learner** [comment]
hey, I was following the interface gan for my task, it is very good but for sunglasses, they are not well enough. In the interface gan they are using some pre-trained boundaries to do the different types of attribute editing and they are working perfectly. I'm working to train over custom boundaries. So, to train boundaries I need the attribute score, so I exactly don't know what it is.
Can you tell me what it is exactly and how can i get it?
